src/containers/Tag/Tag.tsx

```diff
@@ -34,19 +37,47 @@ export const Tag: React.SFC<TagProps> = ({ match }) => {
   const [description, setDescription] = useState('');
   const [keywords, setKeywords] = useState('');
   const [colorCode, setColorcode] = useState('#0C976D');
+  const [parentId, setParentId] = useState<any>([]);
 
-  const states = { label, description, keywords, colorCode };
-  const setStates = ({ label, description, keywords, colorCode }: any) => {
+  const states = { label, description, keywords, colorCode, parentId };
+  const setStates = ({ label, description, keywords, colorCode, parent }: any) => {
     setLabel(label);
     setDescription(description);
     setKeywords(keywords);
     setColorcode(colorCode);
+    if (parent) {
+      setParentId(getObject(data.tags, [parent.id])[0]);
+    }
   };
 
+  const { data } = useQuery(GET_TAGS);
+
+  if (!data) return <Loading />;
+
+  let tags = [];
+  // remove the self tag from list
+  if (data && match && match.params.id) {
+    tags = data.tags.filter(function (tag: any) {
```

Comment:
```suggestion
    tags = data.tags.filter((tag: any) =>  {
```

We should use ES6 syntax